### PR TITLE
2274 feature concept manuel user registration for admins

### LINKED ISSUE #2274: 🚀 [Feature] Concept manuel user registration for admins
## 🚀 Feature

describe the requirements of "manuel user registration for admins"

### PR DESCRIPTION
**Pull request by clauspeterhuebner**
### Issues

- fixes #2274

### Todo

- [X] desgin frontend with mockups
- [X] database migration
- [X] backend service changes

### CODE REVIEW

**Comment by clauspeterhuebner** [comment]
the flaky test blocks this PR even only document files are changed.
As soon as the PR #2255 is merged the flaky test will be solved.

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**
